pkgs/applications/blockchains/alfis/default.nix

```diff
@@ -1,6 +1,5 @@
-{ stdenv, lib, fetchFromGitHub, rustPlatform, pkg-config
-, withGui ? true, webkitgtk, Cocoa, WebKit
-}:
+{ stdenv, lib, fetchFromGitHub, rustPlatform, pkg-config, withGui ? true
```

Comment:
```suggestion
{ stdenv, lib, fetchFromGitHub, rustPlatform, pkg-config, withGui ? stdenv.isLinux
```

Reply:
`withGui` is used to determine whether the user wants to turn GUI on for the `alfis` application. It is not connected specifically to linux.

This PR is meant just to _extend_ capabilities of already existing GUI by adding GUI dialogs, and it does not introduce `withGui` — it was already introduced earlier to make two separate derivations (`alfis` and `alfis-nogui`).

Reply:
> withGui is used to determine whether the user wants to turn GUI on for the alfis application. It is not connected specifically to linux.

Yeah, you are right.